cmd/shell.go

```diff
@@ -255,10 +255,12 @@ type CAConfig struct {
 }
 
 // PAConfig specifies how a policy authority should connect to its
-// database, and what policies it should enforce.
+// database, what policies it should enforce, and what challenges
+// it should offer.
 type PAConfig struct {
 	DBConnect              string
 	EnforcePolicyWhitelist bool
+	ChallengeTypes         []string
```

Comment:
Why not just use a set of `bool`s? It's not as efficient but it seems this approach creates some ambiguity in what challenge types *are correct* for this field, instead of just do I want to activate/disable X.